base/strings/string.jl

```diff
@@ -104,6 +104,27 @@ function prevind(s::String, i::Integer)
     @inbounds while j > 0 && is_valid_continuation(codeunit(s,j))
         j -= 1
     end
+
```

Comment:
Unrelated change. (Below you also have an empty line which isn't there in other functions.)

Reply:
fixed